scripts/aws-clean-up.js

```diff
@@ -0,0 +1,51 @@
+const AWS = require('aws-sdk');
```

Comment:
General comment over this file: the overall flow is correct but it's not a non-blocking workflow. Do you think we should make sure exceptions can be collected and handled at last, so that we delete as much resources as possible?

Reply:
This is because sometimes in CFN, when rollback failed to delete a resource, this stack can never succeed in deleting the resource. I have a bunch of stale stacks in my account... even cannot be deleted from the web UI, so likely a bug from the service. But in this case, we should not block the flow to delete other resources.